ragengine/config.py

```diff
@@ -0,0 +1,15 @@
+# config.py
+import os
+
+EMBEDDING_TYPE = os.getenv("EMBEDDING_TYPE", "local")
+EMBEDDING_URL = os.getenv("EMBEDDING_URL")
+
+INFERENCE_URL = os.getenv("INFERENCE_URL", "http://52.190.41.209/chat")
```

Comment:
why hardcode an ip here?

Reply:
This IP here is a dummy default incase `INFERENCE_URL` environment variable is not found

Reply:
I'll make this more clear

Reply:
If it is a dummy, just use "localhost" instead of ip.

Reply:
Add a comment here mentioning that InferenceURL is specified in RAGEngine CR and expose to pod via pod env.